src/bci_build/package/keadhcp/README.md.j2

```diff
@@ -0,0 +1,36 @@
+# {{ image.pretty_name }} container image
+
+{% include 'badges.j2' %}
+
+ {{ image.description }}
+
+## How to use this container image
+
+The container image expects dhcp configuration file in the path /etc/kea.
+To run dhcp4 for the configuration provided in the directory /etc/kea
```

Comment:
Please also add the expected name of the configuration file
```suggestion
The container image expects configuration file in the directory `/etc/kea`.
Execute the following command to run DHCP using the configuration provided in the directory `/etc/kea`:
```

Reply:
fixed.